rust/cli/src/main.rs

```diff
@@ -560,9 +561,16 @@ struct AddressWithColumns {
     columns: Vec<ColumnWithDetails>,
 }
 
-fn address_to_source(address_with_columns: AddressWithColumns) -> ProjectFileSource {
+fn address_to_source(
+    address_with_columns: AddressWithColumns,
+    source_name_prefix: &Option<String>,
```

Comment:
nit: I am not sure I am sure about this type, not sure a pointer to an Option which owns a string quite makes sense in this context. xx

```suggestion
    source_name_prefix: &Option<&str>, // or just source_name_prefix: &str
```